camp/mgetachew/src/main/java/frc/robot/commands/Backward.java

```diff
@@ -0,0 +1,45 @@
+// Copyright (c) FIRST and other WPILib contributors.
+// Open Source Software; you can modify and/or share it under the terms of
+// the WPILib BSD license file in the root directory of this project.
+
+package frc.robot.commands;
+
+import frc.robot.subsystems.MatthewsCoolMotor;
+import edu.wpi.first.wpilibj2.command.CommandBase;
+
+/** An example command that uses an example subsystem. */
+public class Backward extends CommandBase {
+  @SuppressWarnings({"PMD.UnusedPrivateField", "PMD.SingularField"})
```

Comment:
Not sure we want to get in the practice of suppressing warnings.  Lets bring them to the surface and then when we need them we put a comment in the code that describes why the warning needs to be suppressed.